src/message/signed_message.rs

```diff
@@ -82,6 +82,21 @@ impl SignedMessage {
             cid::Cid::from_cbor_blake2b256(self).expect("message serialization is infallible")
         }
     }
+
+    /// Returns the length of the chain message in bytes.
+    pub fn chain_length(&self) -> anyhow::Result<usize> {
```

Comment:
How about `message_len` or `chain_message_len`?  `chain_length` sounds like number of elements in a chain